src/Facade/ResourceBuilders/ResourceBuildingRetailerDemListActivityVisitor.cs

```diff
@@ -8,12 +10,23 @@ public class ResourceBuildingRetailerDemListActivityVisitor : IRetailerDemListAc
     {
         public RetailerDemListActivityResource Visit(UpdateLastReviewedOnActivity activity)
         {
+            if (activity.LastReviewedOn != null)
+            {
+                return new UpdateLastReviewedOnActivityResource
+                           {
+                               ActivityType = activity.GetType().Name,
+                               UpdatedByUri = activity.UpdatedByUri,
+                               ChangedOn = DateTime.SpecifyKind(activity.ChangedOn, DateTimeKind.Utc).ToString("o"),
+                               LastReviewedOn = DateTime.SpecifyKind((DateTime)activity.LastReviewedOn, DateTimeKind.Utc).ToString("o"),
```

Comment:
I'd take out the if statement and replace this with something like activity.LastReviewedOn.HasValue ? DateTime.SpecifyKind(...) : null;